### PR TITLE
Adds a basic CI for omuse-framework and sub-packages.

### PR DESCRIPTION
**Pull request by merijn**
Runs tests on both gfortran-9 and gfortran-10 to check that the stricter checks in gfortran-10 don't break anything.

### CODE REVIEW

File: src/omuse/community/qgmodel/Makefile [hunk 1]
```diff
@@ -43,4 +43,4 @@ qgmodel_worker: worker_code.f90 $(CODELIB) $(OBJS)
 	$(MPIFC) $(FCFLAGS) $(FS_FLAGS) $< $(OBJS) $(CODELIB) $(FS_LIBS) $(LDFLAGS) -o $@
 
 %.o: %.f90
-	$(MPIFC) $(FCFLAGS) -c -o $@ $<
+	$(MPIFC) $(FCFLAGS) -std=legacy -c -o $@ $<
```
Comment (ipelupessy) [inline src/omuse/community/qgmodel/Makefile:46]: this might be better FCFLAGS+=...

File: src/omuse/community/qgmodel/src/Makefile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 MPIFC ?= mpif90
-FORTRAN= $(FC)
+FORTRAN=$(FC)
+FCFLAGS+=-std=legacy
```
Comment (ipelupessy) [inline src/omuse/community/qgmodel/src/Makefile:3]: then this is not necessary

**Comment by ipelupessy** [comment]
looks ok

File: src/omuse/community/qgmodel/Makefile (hunk 1 above)
Comment (merijn) [reply src/omuse/community/qgmodel/Makefile:46]: That's actually intentional in the (naive?) optimistic hope that the Fortran we generate in amuse/omuse starts complying with an actual Fortran standard so we can remove this one.

Comment (ipelupessy) [reply src/omuse/community/qgmodel/Makefile:46]: ah ok..is there anything in the interface.f90 that is not standard? the generated worker.f90 should already be! (or at least i did some fixes there)

Comment (merijn) [reply src/omuse/community/qgmodel/Makefile:46]: I'm not sure, but it was failing with `-std=f95` and `-std=f2003` and I didn't look into that yet. I wanted to get the tests and new versioning done first, before bothering with that. The tests currently also use the PYPI version of amuse, so if there's fixing in amuse that aren't on PYPI yet, then that'd explain the difference.

Comment (ipelupessy) [reply src/omuse/community/qgmodel/Makefile:46]: I think pypi is up to date (but @rieder might know better)

Comment (rieder) [reply src/omuse/community/qgmodel/Makefile:46]: Pypi is up to date for amuse as of yesterday

**Comment by merijn** [comment]
ugh...broke the PR. Let me reopen it.

**Comment by merijn** [comment]
Unbroken PR in: https://github.com/omuse-geoscience/omuse/pull/47

**Pull request closed without merging**